json/i_want_go_home.json

```
@@ -0,0 +1,7 @@
+{
+  "i": {
+    "want": {
+      "go": "home"
+    }
+  }
```

```suggestion
  "i": {
    "want":  "go home"
  }
```
집에 간다는 것을 강조한다는 의미에서 이런 건 어떨까요?

---

좋은 것 같습니다!

추가적으로, home을 배열로 감싸 여러 장소를 의미할 수 있게 하는 것은 어떨까요?

```suggestion
  "i": {
    "want": {
      "go": ["home"]
    }
  }
```

---

좋아 보입니다!